test/ClangImporter/pch-bridging-header-deps.swift

```diff
@@ -16,10 +16,10 @@
 
 print(app_function(1))
 
-// CHECK-DEPS: pch-bridging-header-deps.o : {{.*}}SOURCE_DIR/test/ClangImporter/Inputs/app-bridging-header-to-pch.h {{.*}}SOURCE_DIR/test/ClangImporter/Inputs/chained-unit-test-bridging-header-to-pch.h
+// CHECK-DEPS: pch-bridging-header-deps.o : {{.*}}{{/|\\}}test{{/|\\}}ClangImporter{{/|\\}}Inputs{{/|\\}}app-bridging-header-to-pch.h {{.*}}{{/|\\}}test{{/|\\}}ClangImporter{{/|\\}}Inputs{{/|\\}}chained-unit-test-bridging-header-to-pch.h
 
 // CHECK-SWIFTDEPS: depends-external:
-// CHECK-SWIFTDEPS: - "SOURCE_DIR/test/ClangImporter/Inputs/app-bridging-header-to-pch.h"
-// CHECK-SWIFTDEPS: - "SOURCE_DIR/test/ClangImporter/Inputs/chained-unit-test-bridging-header-to-pch.h"
+// CHECK-SWIFTDEPS: - "{{.*}}{{/|\\\\}}test{{/|\\\\}}ClangImporter{{/|\\\\}}Inputs{{/|\\\\}}app-bridging-header-to-pch.h"
+// CHECK-SWIFTDEPS: - "{{.*}}{{/|\\\\}}test{{/|\\\\}}ClangImporter{{/|\\\\}}Inputs{{/|\\\\}}chained-unit-test-bridging-header-to-pch.h"
```

Comment:
Why drop the SOURCE_DIR?

Reply:
Excellent question!  The reason for that is because the deps file "escapes" the `:` breaking the `SOURCE_DIR` parameter.  Thinking more about this though, I think that I would rather that the dep file not have the escaped slash.  Any pointers on where to go spelunking for that behaviour?

Reply:
The deps file is a valid YAML file, so I think it has to escape the slash, sorry.

Reply:
Hmm, I don't think that we need to escape it if it is quoted though, right?  The same problem exists for the dependency info too.

Reply:
https://yaml.org/spec/1.2/spec.html#id2786942

I don't think I want to run the risk of switching to single-quoted strings just to make this prettier.